src/app/advancedsearch/advancedsearch.component.spec.ts

```diff
@@ -50,7 +51,8 @@ describe('AdvancedsearchComponent', () => {
         FooterNavbarComponent,
         AboutComponent,
         ContactComponent,
-        ModalComponent
+        ModalComponent,
+        TopbarComponent
```

Comment:
Actually, there is nothing like you have to import the every component into each spec file (of course if you do so you just increase the tests time), you just have to import all components on which the tests depend.